src/EntityEmbedDisplay/EntityEmbedDisplayManager.php

```diff
@@ -98,4 +98,24 @@ public function getDefinitionOptionsForEntity(EntityInterface $entity) {
     }, $definitions);
   }
 
+  /**
+   * Provides a list of plugins that can be used for a certain entity type.
+   *
+   * @param string $entity_type
+   *   The entity type id.
+   *
+   * @return array
+   *   An array of valid plugin labels, keyed by plugin ID.
+   */
+  public function getDefinitionOptionsForEntityType($entity_type) {
+    $definitions = $this->getDefinitions();
+    $valid_definitions = array();
+    foreach ($definitions as $id => $definition) {
+      if ($definition['entity_types'] == FALSE || in_array($entity_type, $definition['entity_types'])) {
```

Comment:
I don't think that'll work. `FALSE` is special in the sense that if entity_types is False then it means that its applicable to all entity types. Changing it to `empty()` would imply that it applies to no entity type which is not what we want here.